src/hotspot/cpu/aarch64/relocInfo_aarch64.hpp

```diff
@@ -29,8 +29,9 @@
   // machine-dependent parts of class relocInfo
  private:
   enum {
-    // Relocations are byte-aligned.
-    offset_unit        =  1,
+    // AArch64 instructions size is 4 bytes.
+    // Two lowest offset bits can always be discarded.
```

Comment:
```suggestion
    // AArch64 instructions are always 4 bytes long and 4-aligned, so the
    // two lowest offset bits can always be discarded.
```

Reply:
@theRealAph Thank you for reviewing and comments.